python/ql/test/experimental/dataflow/coverage/test.py

```diff
@@ -326,22 +326,22 @@ class C:
 
 
 def test_attribute_reference():
-    SINK(C.a)  # Flow missing
+    SINK(C.a) #$ MISSING:flow="SOURCE, l:325 -> C.a" MISSING:flow="'source', l:20 -> C.a"
```

Comment:
```suggestion
    SINK(C.a) #$ MISSING:flow="SOURCE, l:325 -> C.a" MISSING:flow="'source', l:20 -> C.a"
    c = C()
    SINK(c.a) #$ MISSING:flow="SOURCE, l:325 -> C.a" MISSING:flow="'source', l:20 -> C.a"
```

however, I'm fairly scared about messing up line-numbers now, and the pain of fixing them all up.

Reply:
All absolute line numbers are now gone, so we can do this :-D

Reply:
Oh, except in MISSING annotations in that very file...